install.sh

```diff
@@ -251,12 +263,14 @@ main() {
 	# releases in order to download and unpack the right release.
 	CACHE_DIR=$(echo_cache_dir)
 	STANDALONE_INSTALL_PREFIX=${STANDALONE_INSTALL_PREFIX:-/usr/local}
+	TERRAFORM_INSTALL_PREFIX=${TERRAFORM_INSTALL_PREFIX:-/usr/local}
 	STANDALONE_BINARY_NAME=${STANDALONE_BINARY_NAME:-coder}
 	VERSION=${VERSION:-$(echo_latest_version)}
 	# These can be overridden for testing but shouldn't normally be used as it can
 	# result in a broken coder.
 	OS=${OS:-$(os)}
 	ARCH=${ARCH:-$(arch)}
+	TERRA_ARCH=${TERRA_ARCH:-$(terra_arch)}
```

Comment:
Since `TERRAFORM_` has been used above, I think for consistency we should have them all be `TERRAFORM_`.